resources/config.yml

```diff
@@ -1,4 +1,4 @@
-#  Use the § symbol for color coding
+# Use the § symbol for color coding
```

Comment:
This wasn't important to mention in that pull request thing, but I remove a single space from the config file because it bothered me.